src/components/Explore/TokenDetails/TokenDetail.tsx

```diff
@@ -5,14 +5,18 @@ import CurrencyLogo from 'components/CurrencyLogo'
 import { VerifiedIcon } from 'components/TokenSafety/TokenSafetyIcon'
 import TokenSafetyModal from 'components/TokenSafety/TokenSafetyModal'
 import { getChainInfo } from 'constants/chainInfo'
+import { SupportedChainId } from 'constants/chains'
 import { checkWarning } from 'constants/tokenSafety'
+import { gql } from 'graphql-request'
 import { useCurrency, useIsUserAddedToken, useToken } from 'hooks/Tokens'
 import { useAtomValue } from 'jotai/utils'
 import { darken } from 'polished'
-import { useCallback } from 'react'
+import { ReactElement, useCallback } from 'react'
 import { useState } from 'react'
-import { ArrowLeft, Copy, Heart } from 'react-feather'
+import { ArrowLeft, Copy, Heart, TrendingUp } from 'react-feather'
+import { usePreloadedQuery } from 'react-relay'
 import { Link, useNavigate } from 'react-router-dom'
+import { CHAIN_SUBGRAPH_URL } from 'state/data/slice'
```

Comment:
The current graphql URLs in redux are for querying a separate set of data specific to Pools, it has no use here. Relay hook calls with automatically use the correct url once that's setup, so no need to worry about urls here